src/openms/include/OpenMS/FORMAT/Base64.h

```diff
@@ -165,6 +168,23 @@ namespace OpenMS
       UInt32 i;
     };
 
+    
+    const simde__m128i mask1 = simde_mm_set1_epi32(0x3F000000);//00111111 00000000 00000000 00000000
+    const simde__m128i mask2 = simde_mm_set1_epi32(0x003F0000);//00000000 00111111 00000000 00000000
+    const simde__m128i mask3 = simde_mm_set1_epi32(0x00003F00);//00000000 00000000 00111111 00000000
+    const simde__m128i mask4 = simde_mm_set1_epi32(0x0000003F);//00000000 00000000 00000000 00111111
+
+
+
+    const simde__m128i allA = simde_mm_set1_epi8('A');
+    const simde__m128i alla = simde_mm_set1_epi8('a' - 26);
+    const simde__m128i all0 = simde_mm_set1_epi8('0' - 52);
+    const simde__m128i allPlus = simde_mm_set1_epi8('+');
+    const simde__m128i allSlash = simde_mm_set1_epi8('/');
+
+    const simde__m128i shuffleMask = simde_mm_setr_epi8(2, 2, 1, 0, 5, 5, 4, 3, 8, 8, 7, 6, 11, 11, 10, 9);
+    const simde__m128i shuffleMask2 = simde_mm_setr_epi8(3,2,1,0,7,6,5,4,11,10,9,8,15,14,13,12);
+
```

Comment:
What happens here?

the 4 Blocks of Code for a Mask ok, 
themn you Add Letters and Numbers, but why the + / ?? 
and why just an 'A'

the last step? completly mysterious